examples/plot_bcic_iv_2a_moabb_cropped.py

```diff
@@ -107,19 +108,16 @@
     preload=True,
 )
 
-
-class TrainTestBCICIV2aSplit(object):
-    def __call__(self, dataset, y, **kwargs):
-        splitted = dataset.split('session')
-        return splitted['session_T'], splitted['session_E']
-
+splitted = windows_dataset.split('session')
+train_set = splitted['session_T']
+test_set = splitted['session_E']
```

Comment:
maybe better would be `valid_set` as it is used by skorch for validation? For testing, we can think about different dataset?